challenge-manager/challenge-tree/tree.go

```diff
@@ -59,6 +61,8 @@ func New(
 		metadataReader:                metadataReader,
 		histChecker:                   histChecker,
 		validatorName:                 validatorName,
+		totalChallengeLevels:          3,
```

Comment:
Temporary until next PRs that unblock the full suite of multiple challenge levels